data/resources/morale affected.txt

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) 2021 by Luke Arndt
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# This file is for configuring how various events affect morale.
+
+# All fields are optional except "morale affected" and "unique id".
+#
+# Record format:
+# "morale affected" "unique id"
+#   "amount divided by crew members" (number; divided by the number of crew on the ship)
+#   "amount per crew member" (number; multiplied by the number of crew on the ship)
+#   "flat amount" (number; a flat morale change, unaffected by crew count)
+#   "fleet message" (string; displayed if the event applies to a fleet)
+#   "parked multiplier" (number; the morale change is multiplied by this for parked ships)
+#   "ship message" (string; displayed if the event applies to a single ship)
+
+
+# Crew members have died on a ship in the fleet
+# This happens when a ship is destroyed and when crew are lost during a capture attempt
+# Multiplied by the number of crew members that died
+
+"morale affected" "death in fleet"
+  "amount divided by crew members" -1
+  "flat amount" -0.1
+  "fleet message" "In response to crew member deaths,"
+
+
+
+# Some of a ship's crew members have been killed
+# This happens when crew are lost during a capture attempt
+# Multiplied by the number of crew members that died
+# This happens in addition to "death in fleet"
+
+"morale affected" "death on ship"
+  "amount divided by crew members" -10
+  "flat amount" -5
+  "ship message" "Angered and saddened by the deaths of their crew members,"
+
+
+
+# The fleet has disabled an enemy ship
+# Multiplied by the cost of the enemy ship
+
+"morale affected" "enemy ship disabled"
+  "flat amount" 0.0000001
+  "fleet message" "Celebrating their victory,"
+
+
+
+# A ship in the player's fleet has been destroyed
+# Crew deaths on that ship are handed by "death in fleet"
+# Multiplied by the cost of the ship
+
+"morale affected" "fleet ship destroyed"
+  "flat amount" -0.000005
```

Comment:
This seems really low, but I assume this is compensated for with fleet deaths?

Reply:
Yes, and it's bigger than it would seem. This value is multiplied by the cost of the ship, which is usually in the millions.